rllib/algorithms/cql/cql_learner.py

```diff
@@ -0,0 +1,20 @@
+from ray.air.constants import TRAINING_ITERATION
+from ray.rllib.algorithms.sac.sac_learner import SACLearner
+from ray.rllib.core.learner.learner import Learner
+from ray.rllib.utils.annotations import override
+from ray.rllib.utils.metrics import ALL_MODULES
+
+
+class CQLLearner(SACLearner):
+    @override(Learner)
+    def build(self) -> None:
+        # We need to call the `super()`'s `build` method here to have the variables
+        # for `alpha`` and the target entropy defined.
+        super().build()
+
```

Comment:
I feel like it's better to add the NEXT_OBS LearnerConnector piece here, what do you think?

The CQLLearner is the component that has this (hard) requirement (meanin it won't work w/o this connector piece), so it should take care of adding it here.
Similar to how it would check, whether the used RLModule fits a certain API, if required.

Reply:
I agree that the Learner is the component that needs the connectors. On the other side it uses the `algorithm.build_learner_connector` to build its connector pipeline. So, I am a bit unsure, where we should generally pack it. 

In this specific case I needed to add it in the `build_learner_connector` b/c the class inheritance was somehow avoiding it otherwise (i.e. the `CQLLearner` had it, but the `TorchCQLLearner`  did not.